source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+It also represents a return on your deployment investment through better-connected users. 
+
+A push proxy is a key technology behind notification transmission.  It enables notifications between the server and the mobile application.
+
+Mattermost provides a self-hosted Push Proxy you can deploy, the Mattermost Push Notification Service (MPNS). This is also available via a hosted-by-mattermost option, or hosted push notification service (HPNS)*.
+
+    **Note:** `Only Mattermost Enterprise Edition E10 <https://about.mattermost.com/pricing/>`_ or higher to receive access to our Hosted Push Notification Service (HPNS)
+
+If you will be using the Mattermost applications via the iOS and Android app stores, the HPNS is all you need.
+
+Our Hosted Push Notification Service offers:
+
+- Access to a publicly-hosted Mattermost Push Notification Service (MPNS).
+- An explicit `privacy policy <https://about.mattermost.com/hpns-privacy/>`_ for the contents of unencrypted messages.
+- Encrypted TLS connections:
+    - between HPNS and Apple Push Notification Services
+    - HPNS and Google’s Firebase Cloud Messaging service
+    - HPNS and your Mattermost server
+- Production-level uptime expectations
+- Compatibility with EMM Providers*
+
+\* *When using our publicly available app store applications and the AppConfig standard.*
+
+Of course, if you would like to host your own version of the MPNS, that is always an option. The `Choosing the Right Deployment Model <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.6zz62c4mus3l>`_ section of this guide, as well as our `Mobile App Admin Documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ are the best places to start.
+
+********************************************
+Enterprise Mobile Management (EMM) Providers
+********************************************
+
+EMM Providers develop software that helps enterprise teams manage secure mobile technology deployments. This includes the use of mobile devices and usage ready applications.
+
+Most large enterprise teams are familiar with `Enterprise Mobile Management <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`_ providers, or EMMs. If this is a new term for you, review `Appendix B <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.pn9oiqr3rebj>`_ for a list of providers and relevant information.
+
+For those taking on larger deployments, we assume you are already using an EMM Provider.
+
+The AppConfig standard is the critical element when deploying Mattermost via an EMM.
+
+AppConfig is newer, more modern approach when compared to App Wrapping.
+
+We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
+
+While “App Wrapping” has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
```

Comment:
```suggestion
While App Wrapping has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
```

Reply:
App wrapping should also be lowercase. May need to find/replace.

Reply:
+1